ui/src/layout/Menu.js

```diff
@@ -80,33 +96,91 @@ const Menu = ({ onMenuClick, dense, logout }) => {
     )
   }
 
-  const subItems = (subMenu) => (resource) =>
-    resource.hasList && resource.options && resource.options.subMenu === subMenu
+  const subItems = (subMenu) => (resource) => {
+    const { hasList, options, name } = resource
+    return hasList && name !== 'playlist' && options?.subMenu === subMenu
```

Comment:
> Also, is this now available in JS: options?.subMenu Cool! Remind me my good old days of Groovy. I could not find it in the JS specification, do you have a link?
Oh yes, this is one of the es6 features - conditional chaining. It's really cool. I remember feeling really satisfied once i learned about this sometime last year(always felt the other method was too long and cumbersome).
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Also here are some new features from the new es7 release:
https://www.youtube.com/watch?v=v2tJ3nzXh8I

Reply:
We have to be sure this is supported by our babel configuration, to avoid creating incompatibilities with older or mobile browsers. Can you please confirm?

Reply:
Yeah. We have already installed the [necessary package](https://babeljs.io/docs/en/babel-plugin-syntax-optional-chaining) for supporting this.

https://github.com/navidrome/navidrome/blob/c7dc3628e2d58265eb817ad7177f917d00dda903/ui/package-lock.json#L495